atc/exec/policy_check.go

```diff
@@ -0,0 +1,57 @@
+package exec
+
+import (
+	"github.com/concourse/concourse/atc"
+	"github.com/concourse/concourse/atc/policy"
+)
+
+//go:generate counterfeiter . PolicyChecker
+
+type PolicyChecker interface {
```

Comment:
nit: this should probably be called `ImagePolicyChecker` so it doesn't get confused with the API policy checker

Reply:
Fixed.